flickr_download/flick_download.py

```diff
@@ -169,6 +178,10 @@ def main():
                         default=None, help='Quality of the picture')
     parser.add_argument('-n', '--naming', type=str, metavar='NAMING_MODE',
                         default='title', help='Photo naming mode')
+    parser.add_argument('-f', '--folder_naming', type=str, metavar='FOLDER_NAMING',
+                        default='%Y/%m/%d', help='Photo folder naming')
```

Comment:
it is not entirely clear from the doc string what this does, could you clarify. It also, defaults to it being on. It needs to be off for backwards compatibility